core/src/models/BiblePhrase.ts

```diff
@@ -35,6 +35,7 @@ export interface IBiblePhraseWithNumbers extends IBiblePhrase, IBibleReferenceVe
 export type PhraseModifiers = {
     indentLevel?: number;
     quoteLevel?: number;
+    line?: number;
```

Comment:
what does this refer to? Doesn't the `line` attribute of osis specify indent level? Can't quite remember

Reply:
ya, however we need the line number to distinguish between different lines on the db-level, since content group information is saved as phrase modifiers. if there are several lines after each other they would be joined to one line in the db-schema.